src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -247,28 +330,29 @@ public void periodic() {
         m_motor.stopMotor();
         break;
       case MANUAL:
-        if (!m_isCalibrated) break;
+        if (!m_isCalibrated) break; //if the elevator is not calibrated don't do anything. 
         updateElevatorStatus();
         m_motor.set(ControlMode.PercentOutput, m_desiredPower);
         break;
       case POSITION:
-        if (!m_isCalibrated) break;
-        updateElevatorStatus();
-        updateClosedLoopSlot();
-        m_motor.set(
-          ControlMode.Position,
-          Conversions.MetersToFalcon(m_desiredPosition, ElevatorConstants.kSpoolCircumference, ElevatorConstants.kGearRatio),
-          DemandType.ArbitraryFeedForward,
-          m_gravityCompensation
+        if (!m_isCalibrated) break; //if the elevator is not calibrated don't do anything. 
+        updateElevatorStatus(); //set the m_status variable to the desired status requested by a command or trigger or something(not completely sure)
```

Comment:
the m_status variable is used so that when `updateClosedLoopSlot()` is called, it knows which PID and FF to set. maybe, `updateClosedLoopSlot()` should just call `updateElevatorStatus()`